src/Features/LanguageServer/Protocol/Handler/Hover/HoverHandler.cs

```diff
@@ -44,21 +45,15 @@ public HoverHandler(ILspSolutionProvider solutionProvider) : base(solutionProvid
             }
 
             var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
-            return new Hover
+
+            // TODO - Switch to markup content once it supports classifications.
+            // https://devdiv.visualstudio.com/DevDiv/_workitems/edit/918138
+            return new VSHover
             {
                 Range = ProtocolConversions.TextSpanToRange(info.Span, text),
-                Contents = new MarkupContent
-                {
-                    Kind = MarkupKind.Markdown,
-                    Value = GetMarkdownString(info)
-                }
+                Contents = new SumType<SumType<string, MarkedString>, SumType<string, MarkedString>[], MarkupContent>(string.Empty),
```

Comment:
what is a "sum type"?

Reply:
tbh... i literally have no idea what the line is saying.  can you translate?

Reply:
This is just passing an empty string into the contents so we use RawContent instead.

The SumType is there basically because there are multiple different types that can be put into Contents - see https://microsoft.github.io/language-server-protocol/specification#textDocument_hover